libs/accounts/recovery-phone/README.md

```diff
@@ -8,4 +8,8 @@ Run `nx build recovery-phone` to build the library.
 
 ## Running unit tests
 
-Run `nx test recovery-phone` to execute the unit tests via [Jest](https://jestjs.io).
+Run `nx test-unit recovery-phone` to execute the unit tests via [Jest](https://jestjs.io).
+
+## Running integration tests
+
+Run `nx test-integration recovery-phone` to execute the unit tests via [Jest](https://jestjs.io).
```

Comment:
Should this say 'execute the integration tests'?

Reply:
😅